libs/design/src/molecules/article/article/article-theme.scss

```diff
@@ -29,10 +25,14 @@
 			h6 {
 				color: $text-color;
 			}
-		}
 
-		> p {
-			color: $text-color;
+			a {
+				color: daff-color($primary);
```

Comment:
I'm not certain on this one. `a` tags can exist inside of many different tags, including `p` tags. How should we accommodate this?

Reply:
I tried re-enforcing text colors on a[daff-button], but the article style still overrides it. I could add specificity for p > a?